SampleApps/WV2DeploymentVSInstallerSample/README.md

```diff
@@ -14,50 +14,9 @@ products:
-        * Within the `<Commands Reboot="Defer">` and `</Commands>` section, make sure `PackageFile` points to `"MicrosoftEdgeWebview2Setup.exe"` so that the VS installer would be using the Bootstrapper.
-    * For "Download the Evergreen WebView2 Runtime Bootstrapper through link",
-        * Within the `<PackageFiles CopyAllPackageFiles="false">` and `</PackageFiles>` section, un-comment the line `<PackageFile Name="MicrosoftEdgeWebview2Setup.exe" HomeSite="WebViewRuntime" PublicKey="..."/>`, and comment out other lines. Note that the PublicKey for the WebView2 Runtime Bootstrapper may change without notice and we are working on addressing this issue. For now, you may need to replace it with an updated PublicKey.
-        * Within the `<Commands Reboot="Defer">` and `</Commands>` section, make sure `PackageFile` points to `"MicrosoftEdgeWebview2Setup.exe"` so that the VS installer would be using the Bootstrapper.
-    * For "Package the Evergreen WebView2 Runtime Standalone Installer",
-        * Within the `<PackageFiles CopyAllPackageFiles="false">` and `</PackageFiles>` section, un-comment the line `<PackageFile Name="MicrosoftEdgeWebView2RuntimeInstallerX64.exe" />`, and comment out other lines.
-        * Within the `<Commands Reboot="Defer">` and `</Commands>` section, make sure `PackageFile` points to `"MicrosoftEdgeWebView2RuntimeInstallerX64.exe"` so that the VS installer would be using the Standalone Installer.
-        * If you're targeting non-X64 devices, you may also want to edit the `MicrosoftEdgeWebView2RuntimeInstallerX64` filename to reflect the correct architecture.
-1. If you plan to package either the Bootstrapper or the Standalone Installer, [download](https://developer.microsoft.com/microsoft-edge/webview2/) the Bootstrapper or the Standalone Installer and place it under the `WV2DeploymentVSInstallerSample` folder.
-1. Copy the entire `WV2DeploymentVSInstallerSample` folder, and paste it under either,
-    * `Program Files (x86)\Microsoft SDKs\ClickOnce Bootstrapper\Packages\`, or
-    * `<VS Install Dir>\MSBuild\Microsoft\VisualStudio\BootstrapperPackages\` (requires at least VS 2019 Update 7).
-1. Create a Setup Project in Visual Studio. 
-    * In Visual Studio menu, select `File > New > Project`. 
-    * Search for `Setup Project`.
-    ![alt text](../../media/CreateSetupProject.PNG)
-    * Create a Setup Project.
-1. Add WebView2 Runtime as a prerequisite. 
-    * In Visual Studio menu, select `Project > Properties`.
-    * In the Property page, select `Prerequisites..`.
-    ![alt text](../../media/SetupPrerequisites.PNG)
-    * Check `Edge WebView2 runtime`, and un-check other prerequisites. Select `OK`.
-    ![alt text](../../media/SelectPrerequisites.PNG)
-1. Build the Setup project.
+For more information, see [WebView2 Deployment Visual Studio installer](https://docs.microsoft.com/microsoft-edge/webview2/samples/wv2deploymentvsinstallersample).
```

Comment:
add screenshot

Reply:
a good image to copy into the present PR:

https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/samples/wv2deploymentvsinstallersample-images/new-empty-deployment-project-2019.png?branch=pr-en-us-2069

used in rendered PR page section:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/samples/wv2deploymentvsinstallersample?branch=pr-en-us-2069#step-6---create-a-setup-project-in-visual-studio

Reply:
added screenshot
https://github.com/MicrosoftEdge/WebView2Samples/tree/437c01fb862cfff556b86329410171579e97f3e2/SampleApps/WV2DeploymentVSInstallerSample#webview2-deployment-visual-studio-installer